3ds Max/Max2Babylon/Exporter/BabylonExporter.Material.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections.Generic;
 using Autodesk.Max;
 using BabylonExport.Entities;
+using Utilities;
```

Comment:
as previously not sure it is necessary here

Reply:
Yes, we include it for float[] extension methods like float[].AlmostEqualTo